docs/Data/Masking/outages-and-calendar-policy.mdx

```diff
@@ -0,0 +1,17 @@
+# Masked data for local developer use
+
+##### Last updated January 30, 2023
```

Comment:
Mentioned this on a different PR, but I think we can go away from adding this to the docs since if we want to see when / who edited what we can just use git. More info, less manual updates, win win.